README.md

```diff
@@ -6,7 +6,7 @@ Acceptable values are latest or any semantic version string like 1.15.0. Use thi
 ```yaml
 - uses: azure/setup-helm@v1
   with:
-    version: '<version>' # default is latest stable
+    version: v'<version>' # default is latest stable

```

Comment:
This would need code changes as well. And there are multiple places you need to change it.

Reply:
You should add the v to the comment. instead of outside the quotes`''`

Reply:
Add the version to the example above as well `1.15.0` (check for any other instances of versions in this doc)